### PR TITLE
feat(cli): Removed install/uninstall/upgrade commands

### LINKED ISSUE #7904: Core :: Delete install/uninstall/upgrades cmds from CLI
## Technical Details

/\ This is a breaking change /\

In #7903, the cmds have been deprecated. With this ticket, the commands should be finally removed.

## Acceptance Criteria

- [ ] install, uninstall, and upgrades commands are not available in the CLI
- [ ] This is a breaking change

### PR DESCRIPTION
**Pull request by warber**
closes #7904 

```go
$ keptn install
Error: unknown command "install" for "keptn"
Run 'keptn --help' for usage.
$ keptn uninstall
Error: unknown command "uninstall" for "keptn"
Run 'keptn --help' for usage.
$ keptn upgrade
Error: unknown command "upgrade" for "keptn"
Run 'keptn --help' for usage.
```
I also adapted `README.md` accordingly, pls take a look if it's ok.

Further I've removed all kinds of unused functions/constants/vars/etc...

Integration test run: https://github.com/keptn/keptn/actions/runs/2620799900

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,20 @@ Afterwards, install Keptn CLI:
 curl -sL https://get.keptn.sh | bash
 ```
 
-And install Keptn (Note: remove `--use-case=continuous-delivery` in case you don't want to install `helm-service` and `jmeter-service`):
+And install Keptn usint Helm:
```
Comment (mowies) [inline README.md:36]:
```suggestion
And install Keptn using Helm:
```

File: README.md [hunk 2]
```diff
@@ -33,9 +33,20 @@ Afterwards, install Keptn CLI:
...
 ```console
-keptn install --use-case=continuous-delivery
+helm repo add keptn https://charts.keptn.sh && helm repo update
+helm install keptn keptn/keptn \
+-n keptn --create-namespace \
+--wait \
+--set=control-plane.apiGatewayNginx.type=LoadBalancer
```
Comment (mowies) [inline README.md:42]:
```suggestion
--set=apiGatewayNginx.type=LoadBalancer
```

Comment (warber) [reply README.md:42]: took it from here: should we change there as well?https://keptn.sh/docs/quickstart/

Comment (mowies) [reply README.md]: aha yes, i will update that, thanks!

**Pull request merged**